### PR TITLE
[ASM] - Expander - Ev2.2 Api Updates

### PR DESCRIPTION
**Pull request by andrew-paloalto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3268
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-4835

## Description
Adding several new command and capabilities based on new APIs released in Expander 2.2.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by mmhw** [comment]
Hi @efelmandar,
Can you please look here?

**Comment by andrew-paloalto** [comment]
@mmhw @efelmandar this is still a very rough draft, I'll let you know when it's in a more complete state

**Comment by andrew-paloalto** [comment]
@johnnywilkes can you give this a review when you have a chance?

**Comment by johnnywilkes** [comment]
> @johnnywilkes can you give this a review when you have a chance?

@andrew-paloalto , going to add some comments to https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3268

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 1]
```diff
@@ -411,6 +446,240 @@ script:
...
+      description: Category of the Attack Surface Rule
+    - contextPath: ASM.AttackSurfaceRules.remediation_guidance
+      description: Guidance for how to address various ASM risks
+    - contextPath: ASM.AttackSurfaceRules.enabled_status
+      description: Enablement Status of the Attack Surface Rule
+    - contextPath: ASM.AttackSurfaceRules.created
+      description: Creation date of the Attack Surface Rule
+    - contextPath: ASM.AttackSurfaceRules.modified
+      description: Last modification of the Attack Surface Rule
+    description: Fetches Attack Surface Rules related to how Xpanse does assessment.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:480]:
```suggestion
    description: Fetches attack surface rules related to how Cortex Xpanse does assessment.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 2]
```diff
@@ -411,6 +446,240 @@ script:
...
+  - name: asm-tag-asset-assign
+    arguments:
+    - name: asm_id_list
+      required: true
+      description: Comma separated list of asset ids to add tags to.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:485]:
```suggestion
      description: Comma-separated list of asset IDs to add tags to.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 3]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: tags
+      required: true
+      description: 'The name of the tags to apply to supplied assets. '
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:488]:
```suggestion
      description: The name of the tags to apply to supplied assets.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 4]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Assigns tags to a list of assets
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:489]:
```suggestion
    description: Assigns tags to a list of assets.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 5]
```diff
@@ -411,6 +446,240 @@ script:
...
+  - name: asm-tag-asset-remove
+    arguments:
+    - name: asm_id_list
+      description: Comma separated list of asset ids to remove tags from.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:493]:
```suggestion
      description: Comma-separated list of asset IDs to remove tags from.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 6]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: tags
+      description: 'The name of the tags to remove from supplied assets. '
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:495]:
```suggestion
      description: The name of the tags to remove from supplied assets.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 7]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Removes tags from a list of assets
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:496]:
```suggestion
    description: Removes tags from a list of assets.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 8]
```diff
@@ -411,6 +446,240 @@ script:
...
+  - name: asm-tag-range-assign
+    arguments:
+    - name: range_id_list
+      description: Comma separated list of range ids to add tags to.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:500]:
```suggestion
      description: Comma-separated list of range IDs to add tags to.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 9]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: tags
+      description: 'The name of the tags to apply to supplied assets. '
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:502]:
```suggestion
      description: The name of the tags to apply to supplied assets.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 10]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Assigns tags to a list of IP Ranges
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:503]:
```suggestion
    description: Assigns tags to a list of IP ranges.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 11]
```diff
@@ -411,6 +446,240 @@ script:
...
+  - name: asm-tag-range-remove
+    arguments:
+    - name: range_id_list
+      description: Comma separated list of range ids to remove tags from.
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:507]:
```suggestion
      description: Comma-separated list of range IDs to remove tags from.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 12]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: tags
+      description: 'The name of the tags to remove from supplied IP Ranges. '
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:509]:
```suggestion
      description: The name of the tags to remove from supplied IP ranges.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 13]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Removes tags from a list of IP Ranges
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:510]:
```suggestion
    description: Removes tags from a list of IP ranges.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 14]
```diff
@@ -411,6 +446,240 @@ script:
...
+        as "Case IDs'' elsewhere in the API.'
+    - name: description
+      description: String to search for within the incident description field.
+    - name: status
+      auto: PREDEFINED
+      predefined:
+      - new
+      - under_investigation
+      - resolved
+      description: Status to search incidents for
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:524]:
```suggestion
      description: Status to search incidents for.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 15]
```diff
@@ -411,6 +446,240 @@ script:
...
+      description: A date in the format 2019-12-31T23:59:00. Only incidents that were
+        created on or before the specified date/time will be retrieved.
+    - name: gte_creation_time
+      description: A date in the format 2019-12-31T23:59:00. Only incidents that were
+        created on or after the specified date/time will be retrieved.
+    - name: sort_by_creation_time
+      description: Sorts returned incidents by the date/time that the incident was
+        created ("asc" - ascending, "desc" - descending).
+    - name: sort_by_severity
+      description: Sorts returned incidents by the date/time that the incident was
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:535]:
```suggestion
      description: Sorts returned incidents by the severity of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 16]
```diff
@@ -411,6 +446,240 @@ script:
...
+        created ("asc" - ascending, "desc" - descending).
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:536]:
```suggestion

```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 17]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: page
+      description: Page number (for pagination). The default is 0 (the first page).
+    - name: limit
+      description: Maximum number of incidents to return per page. The default and
+        maximum is 100.
+    outputs:
+    - contextPath: ASM.Incident.alert_count
+      description: 'Number of alerts included in the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:544]:
```suggestion
      description: Number of alerts included in the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 18]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Number
+    - contextPath: ASM.Incident.alerts_grouping_status
+      description: 'Whether alert grouping is enabled'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:547]:
```suggestion
      description: Whether alert grouping is enabled.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 19]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.assigned_user_mail
+      description: 'Email of the assigned user'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:550]:
```suggestion
      description: Email of the assigned user.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 20]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.assigned_user_pretty_name
+      description: 'Friendly name of the assigned user'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:553]:
```suggestion
      description: Friendly name of the assigned user.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 21]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.creation_time
+      description: 'Creation timestamp'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:556]:
```suggestion
      description: Creation timestamp.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 22]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Date
+    - contextPath: ASM.Incident.critical_severity_alert_count
+      description: 'Number of critical alerts'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:559]:
```suggestion
      description: Number of critical alerts.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 23]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Number
+    - contextPath: ASM.Incident.description
+      description: 'Description of the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:562]:
```suggestion
      description: Description of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 24]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.high_severity_alert_count
+      description: 'Number of high alerts'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:565]:
```suggestion
      description: Number of high alerts.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 25]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Number
+    - contextPath: ASM.Incident.incident_id
+      description: 'ID of the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:568]:
```suggestion
      description: ID of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 26]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.incident_name
+      description: 'Incident Name'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:571]:
```suggestion
      description: Incident name.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 27]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.incident_sources
+      description: 'Incident Source'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:574]:
```suggestion
      description: Incident source.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 28]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.low_severity_alert_count
+      description: 'Number of low alerts'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:577]:
```suggestion
      description: Number of low alerts.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 29]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Number
+    - contextPath: ASM.Incident.manual_severity
+      description: 'Severity override'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:580]:
```suggestion
      description: Severity override.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 30]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.med_severity_alert_count
+      description: 'Number of medium alerts'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:583]:
```suggestion
      description: Number of medium alerts.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 31]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Number
+    - contextPath: ASM.Incident.modification_time
+      description: 'Modification timestamp'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:586]:
```suggestion
      description: Modification timestamp.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 32]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Date
+    - contextPath: ASM.Incident.notes
+      description: 'Incident notes'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:589]:
```suggestion
      description: Incident notes.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 33]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.original_tags
+      description: 'Tags on the incident at creation time'
+      type: Unknown
+    - contextPath: ASM.Incident.resolve_comment
+      description: 'Resolution comment. This is optional'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:595]:
```suggestion
      description: Resolution comment (optional).
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 34]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.resolved_timestamp
+      description: 'Resolution timestamp'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:598]:
```suggestion
      description: Resolution timestamp.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 35]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    - contextPath: ASM.Incident.severity
+      description: 'Severity of the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:601]:
```suggestion
      description: Severity of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 36]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.starred
+      description: 'Whether the incident has been starred'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:604]:
```suggestion
      description: Whether the incident has been starred.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 37]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Boolean
+    - contextPath: ASM.Incident.status
+      description: 'Status of the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:607]:
```suggestion
      description: Status of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 38]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.tags
+      description: 'Tags on the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:610]:
```suggestion
      description: Tags on the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 39]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: String
+    - contextPath: ASM.Incident.xdr_url
+      description: 'Link to navigate to the incident'
+      type: String
+    - contextPath: ASM.Incident.xpanse_risk_score
+      description: 'Risk score of the incident'
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:616]:
```suggestion
      description: Risk score of the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 40]
```diff
@@ -411,6 +446,240 @@ script:
...
+      type: Unknown
+    description: 'Fetches ASM incidents that match provided filters. Incidents are
+      an aggregation of related alerts. Note: Incident IDs may also be references
+      as "Case IDs'' elsewhere in the API.'
+  - name: asm-update-incident
+    arguments:
+    - name: incident_id
+      required: true
+      description: ID of the incident to modify
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:625]:
```suggestion
      description: ID of the incident to modify.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 41]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: alert_id
+      description: Used for scoping updates such as comments to the alert level
+    - name: assigned_user_mail
+      description: Email address of the user to assign incident to. This user must
+        exist within your Expander instance
+    - name: manual_severity
+      description: Administrator-defined severity for the incident
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:632]:
```suggestion
      description: Administrator-defined severity for the incident.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 42]
```diff
@@ -411,6 +446,240 @@ script:
...
+      predefined:
+      - new
+      - under_investigation
+      - resolved_-_no_longer_observed
+      - resolved_-_no_risk
+      - resolved_-_risk_accepted
+      - resolved_-_contested_asset
+      - resolved_-_remediated_automatically
+      - resolved
+      description: Incident status
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:644]:
```suggestion
      description: Incident status.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 43]
```diff
@@ -411,6 +446,240 @@ script:
...
+    - name: resolve_comment
+      description: Optional resolution comment when resolving the incident
+    - name: comment
+      description: A comment to add to the incident. If an alert_id is supplied it
+        will be prefixed to the comment.
+    outputs:
+    - contextPath: ASM.IncidentUpdate
+      description: Whether the incident update was successful
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:652]:
```suggestion
      description: Whether the incident update was successful.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 44]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Updates a given incident. Can be used to modify the status, severity,
+      assignee, or add comments.
+  - name: asm-update-alerts
+    arguments:
+    - name: alert_id_list
+      description: Comma separated list of integers of the alert ID
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:658]:
```suggestion
      description: Comma-separated list of integers of the alert ID.
```
Perhaps change this to 
Comma-separated list alert IDs.

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 45]
```diff
@@ -411,6 +446,240 @@ script:
...
+      - resolved
+      description: Updated alert status
+    - name: severity
+      auto: PREDEFINED
+      predefined:
+      - low
+      - medium
+      - high
+      - critical
+      description: The severity of the alert
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:678]:
```suggestion
      description: The severity of the alert.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 46]
```diff
@@ -411,6 +446,240 @@ script:
...
+    outputs:
+    - contextPath: ASM.UpdatedAlerts
+      description: IDs of the updated alerts
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:681]:
```suggestion
      description: IDs of the updated alerts.
```

File: Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml [hunk 47]
```diff
@@ -411,6 +446,240 @@ script:
...
+    description: Updates the state of one or more alerts
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml:682]:
```suggestion
    description: Updates the state of one or more alerts.
```

File: Packs/CortexXpanse/ReleaseNotes/1_0_6.md [hunk 48]
```diff
@@ -0,0 +1,27 @@
...
+##### Cortex Xpanse
+Updated the Cortex Xpanse integration to add the following commands:
+- **asm-list-attack-surface-rules**
+- **asm-tag-asset-assign**
+- **asm-tag-asset-remove**
+- **asm-tag-range-assign**
+- **asm-tag-range-remove**
+- **asm-get-incidents**
+- **asm-update-incident**
+- **asm-update-alerts**
```
Comment (ShirleyDenkberg) [inline Packs/CortexXpanse/ReleaseNotes/1_0_6.md:12]:
```suggestion
- ***asm-list-attack-surface-rules***
- ***asm-tag-asset-assign***
- ***asm-tag-asset-remove***
- ***asm-tag-range-assign***
- ***asm-tag-range-remove***
- ***asm-get-incidents***
- ***asm-update-incident***
- ***asm-update-alerts***
```

**Comment by ShirleyDenkberg** [comment]
@andrew-paloalto After approving my comments/edits, please regenerate the integration readme.
@mmhw @efelmandar Doc review completed.

**Review by johnnywilkes — APPROVED** [review]
@andrew-paloalto , effectively did review inside of https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3268

Approved!

**Comment by johnnywilkes** [comment]
@mmhw / @efelmandar , this has been approved by Xpanse.  Please merge when possible.

**Review by mmhw — APPROVED** [review]
Good work!

**Pull request merged**